### PR TITLE
Add support for db.sync_wal()

### PR DESCRIPTION
**Pull request by arthurprs**
Add support for db.sync_wal()

### CODE REVIEW

**Comment by siddontang** [comment]
Thanks @arthurprs 

Please add a test

**Comment by arthurprs** [comment]
Added the test.

Thanks for maintaining the library.

File: librocksdb_sys/build.sh [hunk 1]
```diff
@@ -3,10 +3,14 @@
 set -e
 
 con=1
-if [[ -f /proc/cpuinfo ]]; then
-    con=`grep -c processor /proc/cpuinfo`
+if [ "$MAKE_PARALLELISM" ]; then
```
Comment (siddontang) [inline librocksdb_sys/build.sh]: hi @arthurprs 

Where to use MAKE_PARALLELISM here? I suggest using another PR for this approach :-)

**Comment by ngaut** [comment]
LGTM

File: librocksdb_sys/build.sh (hunk 1 above)
Comment (arthurprs) [inline librocksdb_sys/build.sh]: I had to add this and set the env var to 1 otherwise g++ would just run out of memory in travis.

**Pull request merged**